examples/Rules-class-StoppingMTDCV.R

```diff
@@ -0,0 +1,7 @@
+
+# As example, here is the rule for: 
```

Comment:
see the linter warnings here - just remove whitespace etc then it will disappear